packages/fields/src/fields/slug/style.scss

```diff
@@ -0,0 +1,18 @@
+.fields-controls__slug {
```

Comment:
Does it make sense this classname? Or should we think a better one?

Reply:
Perhaps I'd go with `.field-slug` as a prefix.